src/frontend/utils/fetch.ts

```diff
@@ -2,9 +2,12 @@ import { v4 as uuidv4 } from 'uuid';
 
 import { RessursFeilet, RessursStatus, RessursSuksess } from '../typer/ressurs';
 
+export type Method = 'GET' | 'POST' | 'PUT' | 'PATCH' | 'DELETE' | 'UPDATE';
+
 export const fetchFn = <ResponseData, RequestData>(
     url: string,
-    method: 'GET' | 'POST' | 'PUT' | 'PATCH' | 'DELETE' | 'UPDATE' = 'GET',
+    method: Method,
+    settAutentisert: () => void,
```

Comment:
Er vel dette den gjør?
```suggestion
    settIkkeAutentisert: () => void,
```

Reply:
Ja 😁